client/src/pages/Profile.js

```diff
@@ -1,5 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { ReactComponent as User } from '../assets/user.svg';
+import { Loader } from '../components';
 
-const Profile = () => <h1> Profile </h1>;
+const Profile = () => {
+  const [profile, setProfile] = useState();
+
+  const getProfile = async () => {
+    const { data } = await axios.get('api/v1/profile');
```

Comment:
handle error here by using ```try catch```
``
```suggestion
    try {
      const { data } = await axios.get('api/v1/profile');
      setProfile(data[0]);
    } catch (err) {
      if (err.response) setError(err.response.data.message);
      else setError('Something went error');
    }
  };
```
you can add a state for error
``` const [error, setError] = useState(''); ```